dojson/contrib/marc21/utils.py

```diff
@@ -126,7 +126,22 @@ def split_blob(blob):
     raise StopIteration()
 
 
+def split_stream(stream):
+    """Split the stream using <record.*?>.*?</record> as pattern."""
+    buf = []
```

Comment:
We should make `create_record` to accept the element of `lxml` tree element anyway so we don't create and parse strings around.